price-alert.py

```diff
@@ -1,18 +1,27 @@
 #!/usr/bin/env python3
 
-import os
 import re
-import json
 import time
 import requests
 import smtplib
-import argparse
 import logging
 from copy import copy
```

Comment:
Maybe this one below the `import json` :)

Reply:
fixed